CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 - Feat: Use `Hint` for screenshots ([#1165](https://github.com/getsentry/sentry-dart/pull/1165))
 - Feat: Support custom units for custom measurements ([#1181](https://github.com/getsentry/sentry-dart/pull/1181))
 
+### Enhancements
+
+- Enha: Exceptions not handled by user code should be `unhandled: true` #1186 ([#1186](https://github.com/getsentry/sentry-dart/pull/1186))
```

Comment:
```suggestion
- Exceptions not handled by user code should be `unhandled: true` #1186 ([#1186](https://github.com/getsentry/sentry-dart/pull/1186))
```

We don't need a prefix because we already have the headlines